set.mm

```diff
@@ -644661,29 +644740,97 @@ collection and union ( ~ mnuop3d ), from which closure under pairing
-      ZVDOVIVJVHOZVHHNZOZFNZVLOVJVNOZJNZVNOKNZVPOVQVFOPKQPJQRPPFQRPPPPIQRPREQRP
-      DQRHQPCQZRPZBQRZAVCVELOZVBSZBUHVTVBBLUBWBVSBWBVBWASVSRWAVBUCVBVBWAVRVBUDW
-      AVEUEOZVDUIZVETWDVFTVKVMSEVEMVOVNUFVFTSFVLMPIVDUGSDVEMSHQCVEUGZVRLUEVEUJU
-      KWCWEUSBCDEFVEHIULUMUNCDEFGVEHIJKUOUPUQUTURUTVA $.
+      ( cv wcel wrex wi wal wn wa wss bitri vk vm vn vr vq vp vl cgru wex ancom
+      df-rex biid cpw cuni wral cvv grumnueq ismnu ismnuprim expandan expandexn
+      wb elv albii ) ALBLZMZBUHZNZVFCLZVEMDLZVEMGLZELZMVKVIMOGPVLVEMZVLVJMQOQOI
+      LZVIMVMVNVLMZVLHLZMZFLZVPMVNVRMZJLZVRMKLZVTMWAVJMOKPOJPQOOFPQOOOOIPQOQEPQ
+      ODPQHPOCPZQOZBPQZAVHVEVGMZVFRZBUIWDVFBVGUKWFWCBWFVFWERWCQWEVFUJVFVFWEWBVF
+      ULWEVIUMZVESWGVJSVOVQREVENVSVRUNVJSRFVPNOIVIUORDVENRHPCVEUOZWBWEWHVBBCDEF
+      VEHIUAUBUCVGUPUDUEUFUGUAUBUCUDUEUFUGUQURVCCDEFGVEHIJKUSTUTTVATVD $.
+  $}
+
+  ${
+    inagrud.1 $e |- ( ph -> I e. Inacc ) $.
+    $( Inaccessible levels of the cumulative hierarchy are Grothendieck
+       universes.  (Contributed by Rohan Ridenour, 13-Aug-2023.) $)
+    inagrud $p |- ( ph -> ( R1 ` I ) e. Univ ) $=
+      ( cr1 cfv ctsk wcel wtr cgru cina inatsk syl r1tr grutsk1 sylancl ) ABDEZ
+      FGZPHPIGABJGQCBKLBMPNO $.
+  $}
+
+  ${
+    $d x A $.
+    $( Assuming the Tarski-Grothendieck axiom, every ordinal is contained in an
+       inaccessible ordinal.  (Contributed by Rohan Ridenour, 13-Aug-2023.) $)
+    inaex $p |- ( A e. On -> E. x e. Inacc A e. x ) $=
+      ( con0 wcel cv cina csuc cdif cint wceq wa wss cwina inawina winaon ssriv
+      syl onmindif mpan adantr simpr eleqtrrd wne difss sstri cvv inaprc ssdif0
+      c0 neli wi sucexg ssexg expcom syl5bir mtoi neqned onint sylancr rr-rspce
+      eldifad ) BCZDZBAEZDAFZBGZHZIZVEVCVDVHJZKBVHVDVCBVHDZVIVEVBLVCVJAVEVBVDVE
+      DVDMDVDVBDVDNVDOQPZVEBRSTVCVIUAUBVCVHVEVFVCVGVBLVGUIZUCVHVGDVGVEVBVEVFUDV
+      KUEVCVGVLVCVGVLJZVEUFZDZVEVNUGUJVMVEVFLZVCVOVEVFUHVCVFVNDZVPVOUKBVBULVPVQ
+      VOVEVFVNUMUNQUOUPUQVGURUSVAUT $.
+  $}
+
+  ${
+    $d x y z $.
+    $( Assuming the Tarski-Grothendieck axiom, every set is contained in a
+       Grothendieck universe.  (Contributed by Rohan Ridenour, 13-Aug-2023.) $)
+    gruex $p |- E. y e. Univ x e. y $=
+      ( vz crnk cfv wcel cina wrex cgru con0 rankon inaex ax-mp cr1 wceq simplr
+      cv wa wb syl cwina inawina winaon ad2antrr vex rankr1a mpbird simpr simpl
+      eleqtrrd inagrud rr-rspce rexlimiva ) AQZDEZCQZFZCGZHZUNBQZFZBIZHZUOJZFUS
+      UNKCUOLMUQVCCURUPURFZUQRZVABUPNEZVBVFUTVGOZRZUNVGUTVIUNVGFZUQVEUQVHPVIUPV
+      DFZVJUQSVEVKUQVHVEUPUAFVKUPUBUPUCTUDUNUPAUEUFTUGVFVHUHUJVFUPVEUQUIUKULUMM
+      $.
+  $}
+
+  ${
+    $d x y $.  $d y z w v f i k m n q p l $.  $d y z w u f i k m n r p l $.
+    $( An equivalent of ~ ax-groth using only simple defined symbols.
+       (Contributed by Rohan Ridenour, 13-Aug-2023.) $)
+    rr-groth $p |- E. y ( x e. y /\ A. z e. y A. f ( ~P z C_ y /\ E. w e. y
```

Comment:
If you're not too tired with this MR, I would suggest
`A. f ( ~P z C_ y /\`
-->
`( ~P z C_ y /\ A. f`

Reply:
(basically, choose the smaller scope for quantifiers; same below if applicable)